test/Common.cpp

```diff
@@ -277,4 +312,14 @@ bool loadVMs(CommonOptions const& _options)
 	return true;
 }
 
+void printOptions(std::ostream& _stream, std::string const& _linePrefix, CommonOptions const& _options, std::vector<std::string> const& _filter)
+{
+	_stream << _linePrefix << "Run Settings: " << _options.toString(_filter) << std::endl;
+}
+
+void printOptions(std::ostream& _stream, std::string const& _linePrefix, CommonOptions const& _options)
```

Comment:
It would make more sense to call this something like `printImportantOptions()`. Without the argument I'd actually expect it to print everyting, not some random subset.
```suggestion
void printImportantOptions(std::ostream& _stream, std::string const& _linePrefix, CommonOptions const& _options)
```

Reply:
I have changed it to `printSelectedOptions`.